shell/platform/android/io/flutter/embedding/engine/loader/FlutterLoader.java

```diff
@@ -234,13 +233,20 @@ public void ensureInitializationComplete(
       }
 
       long initTimeMillis = SystemClock.uptimeMillis() - initStartTimestampMillis;
+
+      // TODO(cyanlaz): Remove this when dynamic thread merging is done.
+      // https://github.com/flutter/flutter/issues/59930
+      Bundle bundle = applicationInfo.metaData;
+      boolean use_embedded_view = bundle.getBoolean("io.flutter.embedded_views_preview");
+
       FlutterJNI.nativeInit(
           applicationContext,
           shellArgs.toArray(new String[0]),
```

Comment:
nit: what about passing the flag in `shellArgs`?

Reply:
done